### DISCUSSION TITLE
Align Columns position and size in Events view

### DISCUSSION THREAD

**Discussion by PassivePicasso** [datalust/seq-tickets#1299] [Ideas]
Signal Columns are an excellent way to make log entries more clear.  However sometimes the variability of their size can make it hard to read logs.
It would be great to have the ability to have columns automatically size to their widest element so that the natural @Message column is always aligned on its left edge in addition to each Signal column being aligned.

This would provide some greatly enhanced readability.   I'm all for this being optional, or being configurable in various ways.

**Comment by PassivePicasso** [comment]
An Alternative method to handle this could be to give classes to property columns with the name of the property. 
This way for example, a width could be set for a Username column or an Entity ID if the property column element had "entityid" as a class instead of "tag-0" or "tag-1"

you can get close to the functionallity I mentioned by setting min-width on the property-name class, however this has unintended side effects such as modifying the width of property titles in expanded rows as well.

**Reply by nblumhardt** [reply]
Thanks for the suggestion. We're interested in exploring this more, but it will need to roll into a larger update to the Events screen which we're planning to make in 2022.

In case it helps, this style (applied via _Settings > Instance_) works pretty well for me:

```css
.event-columns .property-name {
    min-width: 120px;
}
```

Best regards,
Nick

**Reply by PassivePicasso** [reply]
I look forward to seeing the improvements, and I'm eagerly awaiting the next release for a few features!

Thanks for the tip, adding .event-columns to my selector fixed the unintended side effects I ran into,
